wscript

```diff
@@ -399,6 +399,10 @@ iconv support use --disable-iconv.",
         'name': 'jpegxl',
         'desc': 'JPEG XL support via libavcodec',
         'func': check_pkg_config('libavcodec >= 59.27.100'),
+    }, {
+        'name': 'avif_muxer',
+        'desc': 'avif support via libavcodec',
+        'func': check_pkg_config('libavcodec >= 59.24.100'),
```

Comment:
Should this be libavformat for consistency with `meson.build`? iirc the JXL discrepancy is a bug.